Rest/Controller/ContentController.php

```diff
@@ -131,6 +136,36 @@ public function getContent($contentIdList, Request $request)
         return new ContentDataValue($content);
     }
 
+    /**
+     * Preparing array of integers based on comma separated integers in string or single integer in string.
+     *
+     * @param string $string list of integers separated by comma character
+     *
+     * @return array
+     *
+     * @throws LogicException When incorrect $list value is given
+     */
+    protected function getIdListFromString($string)
+    {
+        if (is_numeric($string)) {
+            return array($string);
+        }
+
+        if (strpos($string, ',') === false) {
+            throw new LogicException('Integers in %s should have a separator');
```

Comment:
`InvalidArgumentException` is precise in this case. Besides that `%s` should be replaced by some value 😉